notebooks/Community.ipynb

```diff
@@ -0,0 +1,221 @@
+   "source": [
+    "# Read graph\n",
+    "G = nk.readGraph(\"../input/karate.graph\", nk.Format.METIS)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Using the [detectCommunities(G, algo=None, inspect=True)](https://networkit.github.io/dev-docs/python_api/community.html?highlight=detect#networkit.community.detectCommunities) function, we specify `PLM` as the algorithm of choice, and pass the parameters as follows: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Choose and initialse algorithm \n",
+    "plmCommunities = nk.community.detectCommunities(G, algo=nk.community.PLM(G, True,0.02))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The result of community detection is a partition of the node set into disjoint subsets. It is represented by the [Partition](https://networkit.github.io/dev-docs/python_api/networkit.html?highlight=partition#networkit.Partition) data strucure, which provides several methods for inspecting and manipulating a partition of a set of elements (which need not be the nodes of a graph)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"{0} elements assigned to {1} subsets\".format(plmCommunities.numberOfElements(),\n",
+    "                                                    plmCommunities.numberOfSubsets()))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"the biggest subset has size {0}\".format(max(plmCommunities.subsetSizes())))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The contents of a partition object can be written to file in a simple format, in which each line `i` contains the subset id of node `i`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nk.community.writeCommunities(plmCommunities, \"output/communtiesPLM.partition\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## PLP "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Label Propagation algorithm is an algorithm for finding communities in a graph. NetworKit provides a parallel implementation, [PLP(G, updateThreshold,maxIterations)](https://networkit.github.io/dev-docs/python_api/community.html?highlight=plp#networkit.community.PLP). The constructor expects a [networkit.Graph](https://networkit.github.io/dev-docs/python_api/networkit.html?highlight=graph#networkit.Graph) as a mandatory parameter. The parameter `updateThreshold` dictates the number of nodes that have to be changed in each iteration so that a new iteration starts, and `maxIterations` is the maximum number of iterations.\n",
```

Comment:
Write PLP default parameters.

Reply:
The default values of both parameters is `none`, which is not in the Python documentation. How should I express this? Also, should I then I add then it to the documentation?

Reply:
Yes you can add it to the Python documentation and mention it in the notebook.